src/types/form.ts

```diff
@@ -264,7 +264,7 @@ export type UseFormResetField<TFieldValues extends FieldValues> = <
 ) => void;
 
 export type UseFormReset<TFieldValues extends FieldValues> = (
-  values?: DefaultValues<TFieldValues>,
+  values?: DefaultValues<TFieldValues> | UnpackNestedValue<TFieldValues>,
```

Comment:
We suggest the following modifications.
Does this fix not solve the problem?

```suggestion
  values?: UnpackNestedValue<DefaultValues<TFieldValues>,
```